build-system/eslint-rules/html-template.js

```diff
@@ -177,7 +177,7 @@ function create(context) {
     // Void tags are defined at
     // https://html.spec.whatwg.org/multipage/syntax.html#void-elements
     const invalid =
-      /<(?!area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)([a-zA-Z-]+)([\s\n][^>]*)?\/>/gm;
+      /<(?!area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)(((svg)?)([a-zA-Z-])+)([\s\n][^>]*)?\/>/gm;
```

Comment:
Skipping the call to `invalidVoidTag` for `svg` tags would fix this without requiring full regex scans.

Reply:
agreed, and changed to skip the call to invalidVoidTag for svg elements.